docs/niswitch/attributes.rst

```diff
@@ -1215,8 +1235,12 @@ niswitch.Session properties
 
 .. py:attribute:: record_coercions
 
-    Specifies whether the IVI engine keeps a list of  the value coercions it makes for ViInt32 and ViReal64 attributes.   You call niSwitch_GetNextCoercionRecord to extract and delete the oldest  coercion record from the list.
-    The default value is VI_FALSE.   Use the  niSwitch_InitWithOptions function to override this value.
+    Specifies whether the IVI engine keeps a list of  the value coercions it makes for ViInt32 and ViReal64 attributes.   You call :py:meth:`niswitch.Session.GetNextCoercionRecord` to extract and delete the oldest  coercion record from the list.
+    The default value is VI_FALSE.   Use the  :py:meth:`niswitch.Session.InitWithOptions` function to override this value.
+
+
+
+    .. note:: One or more of the referenced functions are not in the Python API for this driver.
```

Comment:
Is this line added in this review? Do you always add it or it depends on the specific metadata?

Reply:
It is added based on the metadata. If we see something that won't be in the API, add it. Similar text for attributes and enums.

Reply:
In this same PR?

Reply:
It is a consequence of fixing the backslashes. It should have been there before.

It is added dynamically during add all.